ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteImporter.java

```diff
@@ -258,6 +258,9 @@ private static void createDanglingLine(UcteLine ucteLine, boolean connected,
                 .setQ0(q0)
                 .setUcteXnodeCode(xnode.getCode().toString())
                 .add();
+        if (ucteLine.getElementName() != null && !ucteLine.getElementName().isEmpty()) {
+            dl.addAlias(String.format("%s %s %s", ucteLine.getId().getNodeCode1().toString(), ucteLine.getId().getNodeCode2().toString(), ucteLine.getElementName()));
```

Comment:
Is really what you want? Maybe the elementName should be an alias, maybe "VL1 VL2 order" should be another one.
Maybe it could be nice to create a function for that

Reply:
As "node1/node1/order" is already the id, no need of it as an alias.
The element name cannot be an alias itself, cause TSOs usually use it as a kind of order code (you can have same element name for different lines)

Reply:
I know, it is the reason why I suggest "VL1 VL2 co" to complement the "node1 node2 co". If I remember, that was a way to reference equipments in UCTE process. Maybe it's not the case anymore.

Reply:
Good remark, it's true that sometimes the node code is replaced by * in the input.